lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -232,15 +243,27 @@ SWAP_BP_FOR_OPCODE:
                 break;
             }
             case INTERPRETER_OPCODE::ExtendedMediumLayoutPrefix:
+            case INTERPRETER_OPCODE::DblExtendedMediumLayoutPrefix:
             {
 #ifndef INTERPRETER_ASMJS  // Asmjs doesn't have any extended opcodes for now, remove that case
```

Comment:
Hmm... I didn't know this was here, but it doesn't seem valid. I think this will cause bug in interpreter dispatch on SIMD in asm.js when you have medium/large bytecode register number.